### PR TITLE
Update NAMING.md

### PR DESCRIPTION
**Pull request by el00ruobuob**
Update proposal according to the community voting for Lithium Luna.

### CODE REVIEW

File: NAMING.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
 # Monero Release Naming Convention
 
-Monero's naming convention of a chemical element, and a star or constellation or Messier object, and was decided on via a discussion on the official forum in 2015. Alliteration or assonance is also required, for maximum awesomeness. As an example, the first Monero release following this convention was named "Hydrogen Helix".
+Monero's naming convention of a chemical element, and celestial object, and was decided on via a discussion on the official forum in 2015 and a community rush to the moon in 2018. Alliteration or assonance is also required, for maximum awesomeness. As an example, the first Monero release following this convention was named "Hydrogen Helix".
 
 Below is a list of elements and galactic bodies, as prepared by palexander.
 
-| Elements | Galaxies, Messier Objects | 
+| Elements | celestial Objects |
```
Comment (QuickBASIC) [inline NAMING.md]: /s/celestial/Celestial

**Comment by el00ruobuob** [comment]
Updated.

File: NAMING.md (hunk 1 above)
Comment (el00ruobuob) [reply NAMING.md]: Done

**Comment by luigi1111** [comment]
What's with the "community rush to the moon"? Also, please squash.

**Comment by el00ruobuob** [comment]
I was refering to everybody wanting to call 0.12 Lithium Luna. I corrected it for clarity.

**Comment by luigi1111** [comment]
You did the opposite of squashing. :)

**Comment by el00ruobuob** [comment]
I'm lost. Isn't squashing grouping everything in a single commit?

**Comment by luigi1111** [comment]
Maybe I'm blind.

**Pull request merged**